client/types/utils.ts

```
@@ -0,0 +1,9 @@
+export type SerializeDates<T> = {
+  [K in keyof T]: T[K] extends Date
+    ? string
+    : T[K] extends (infer U)[] | readonly (infer U)[]
```

Biome (1.9.4)

[error] Shouldn't redeclare 'U'. Consider to delete it or rename it.

'U' is defined here:


(lint/suspicious/noRedeclare)